src/io/forsta/ccsm/database/ContactDb.java

```diff
@@ -351,6 +351,15 @@ public Cursor getById(String id) {
     return null;
   }
 
+  public boolean checkForTag(String tag) {
+    try {
+      return doesRowExist(TABLE_NAME, SLUG, tag);
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return false;
+  }
+
```

Comment:
I should make or find a method that returns the user ID when given the tag as a search criteria

Reply:
I think getUserByTag would be a better way to handle this. There is already a method called getUserByAddress, so just use that as an example. Return a ForstaUser object. It will have the user's ID.